src/config.rs

```diff
@@ -54,6 +54,7 @@ pub struct SpotifyConfig {
     #[serde(default)]
     pub extra_env_vars: Vec<String>,
     pub download_attempts: Option<usize>,
+    pub post_update_exec_hook: Option<String>,
```

Comment:
```suggestion
    #[serde(default)]
    pub post_update_exec_hook: Vec<String>,
```
When thinking of commands as lists, users could pass arguments here too (without introducing the concept of "shell").

It should be marked `default` for serde so existing configuration doesn't break.

Reply:
I'm sort of on the fence to complicate this feature for the end-user, I'd like to keep it as simple as possible for now. Are you okay with keeping it this way for the first iteration. If needed, we can always implement this later.

For the `default` marking: as-is it should not be needed as the entry is wrapped in an `Option<...>`.